http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/RuntimeRoutes.scala

```diff
@@ -478,58 +481,35 @@ object RuntimeRoutes {
     "blockSize"
   )(x => (x.name, x.size, x.diskType, x.blockSize))
 
-  // we're reusing same `GetRuntimeResponse` in LeonardoService.scala as well, but we don't want to encode this object the same way the legacy
-  // API does
-  implicit val getRuntimeResponseEncoder: Encoder[GetRuntimeResponse] = Encoder.forProduct21(
-    "id",
-    "runtimeName",
-    "googleProject",
-    "serviceAccount",
-    "asyncRuntimeFields",
-    "auditInfo",
-    "runtimeConfig",
-    "proxyUrl",
-    "status",
-    "labels",
-    "jupyterUserScriptUri",
-    "jupyterStartUserScriptUri",
-    "errors",
-    "userJupyterExtensionConfig",
-    "autopauseThreshold",
-    "defaultClientId",
-    "runtimeImages",
-    "scopes",
-    "customEnvironmentVariables",
-    "diskConfig",
-    "patchInProgress"
-  )(x =>
-    (
-      x.id,
-      x.clusterName,
-      x.googleProject,
-      x.serviceAccountInfo,
-      x.asyncRuntimeFields,
-      x.auditInfo,
-      x.runtimeConfig,
-      x.clusterUrl,
-      x.status,
-      x.labels,
-      x.jupyterUserScriptUri,
-      x.jupyterStartUserScriptUri,
-      x.errors,
-      x.userJupyterExtensionConfig,
-      x.autopauseThreshold,
-      x.defaultClientId,
-      x.clusterImages,
-      x.scopes,
-      x.customClusterEnvironmentVariables,
-      x.diskConfig,
-      x.patchInProgress
+  // can't use Encoder.forProductX because there are 23 fields :(
+  implicit val getRuntimeResponseEncoder: Encoder[GetRuntimeResponse] = Encoder.instance { x =>
```

Comment:
would be nice to somehow inform users about when we'll remove the deprecated params and have an actual time frame when we plan to do that